src/apps/applets/api/applets.py

```diff
@@ -1,52 +1,63 @@
 from fastapi import Body, Depends
 
+import apps.applets.domain as domain
```

Comment:
Well, actually we can do 
```suggestion
from apps.applets import domain
```

Reply:
okay